x/staking/keeper/delegation.go

```diff
@@ -104,7 +107,10 @@ func (k Keeper) SetDelegation(ctx sdk.Context, delegation types.Delegation) {
 
 // RemoveDelegation removes a delegation
 func (k Keeper) RemoveDelegation(ctx sdk.Context, delegation types.Delegation) error {
-	delegatorAddress := sdk.MustAccAddressFromBech32(delegation.DelegatorAddress)
+	delegatorAddress, err := k.authKeeper.StringToBytes(delegation.DelegatorAddress)
+	if err != nil {
+		panic(err)
```

Comment:
Why not return the error here? It may be weird however.

Reply:
recreates current behavior, to add error would break the api and ballon the size of the pr. wanted to keep it small